src/contest.c

```diff
@@ -1636,7 +1636,9 @@ static void Task_ShowMoveSelectScreen(u8 taskId)
         moveNameBuffer = StringCopy(moveNameBuffer, GetMoveName(move));
 
         FillWindowPixelBuffer(i + MOVE_WINDOWS_START, PIXEL_FILL(0));
-        Contest_PrintTextToBg0WindowAt(i + MOVE_WINDOWS_START, moveName, 5, 1, FONT_NARROW);
+	// 5 + 7*8 because there are 7 tiles and about 5 extra pixels
+	// available.
```

Comment:
Spaces instead of tabs. Also, would `TILE_WIDTH` or similar make sense to be used here instead of 8?
```suggestion
        // 5 + 7*8 because there are 7 tiles and about 5 extra pixels available.
```